pkg/sdk/dbpopulator/populator.go

```diff
@@ -137,9 +137,14 @@ CALL {
   WITH input, parameters
   UNWIND keys(parameters) as name
     CREATE (parameter: InputParameter:unpublished {
```

Comment:
only one thing can be specified, either `jsonSchema` or `typeRef`. You need to check which one was specified and populate a proper data.